src/aks-preview/azext_aks_preview/__init__.py

```diff
@@ -16,7 +16,11 @@ class ContainerServiceCommandsLoader(AzCommandsLoader):
 
     def __init__(self, cli_ctx=None):
         from azure.cli.core.commands import CliCommandType
-        register_resource_type('latest', CUSTOM_MGMT_AKS_PREVIEW, '2021-05-01')
+        register_resource_type(
+            "latest",
+            CUSTOM_MGMT_AKS_PREVIEW,
+            SDKProfile("2021-05-01", {"container_services": "2017-07-01"}),
```

Comment:
May I ask why this api-version for `container_services` uses an older one?

Reply:
The "container_services" operations are only used to implement ```aks get-version``` and not updated for a long time (which is the same in azure-cli). Other ```aks xxx``` operations are supported by "managed_clusters" and "agent_pools" operations.

Reply:
OK, got it~